source/common/tracing/zipkin/util.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+namespace Zipkin {
+
+/**
+ * Utility class with a few convenient methods
+ */
+class Util {
```

Comment:
this class looks like a sink for all possible utility methods :-)
I think some of those methods are already in existing code base, can you reuse (i point to some examples above)